code/__DEFINES/melee.dm

```diff
@@ -8,3 +8,4 @@
 #define MARTIALART_KRAVMAGA "krav maga"
 #define MARTIALART_CQC "CQC"
 #define MARTIALART_PLASMAFIST "plasma fist"
+#define MARTIALART_MONKEYSTYLE "monkey style"
```

Comment:
I don't see this mentioned anywhere.

Reply:
ah yeah i ended up not using it, good catch